src/main/kotlin/additionparser/core/Splitter.kt

```diff
@@ -0,0 +1,16 @@
+package additionparser.core
+
+class Splitter(private val token: String, private val customDelimiter: String = "") {
+    fun split(): List<String> {
+        return token.split(Regex(getPattern()))
+    }
+
+    private fun getPattern(): String {
+        val strBuilder = StringBuilder()
+        strBuilder.append("[,;")
+        strBuilder.append(customDelimiter)
+        strBuilder.append("]")
+
+        return strBuilder.toString()
```

Comment:
이 코드를 처음보는 사람도 이해할 수 있도록 ,`, `;`, `[`, `]`를 상수로 추출해서 의도를 드러내주면 좋을 것 같네요. 😃

추가적으로 아래와 같이 Collection의 확장함수인 `joinToString`를 활용해볼 수도 있을 것 같네요. 😃

```suggestion
        return listOf(",", ";", customDelimiter).joinToString("", "[", "]")
```

Reply:
전달해주신 코드가 더 간결할거 같네요. joinToString을 사용한 코드로 치환하였습니다.